src/utils/IPFS.js

```diff
@@ -54,6 +54,10 @@ class IPFS {
       return null
     })
   }
+
+  getAsIs (hash) {
+    return this._api.get(hash)
```

Comment:
Picked from another commit by mistake.
We need to implement a way to IPFS.get() uses JSON.parse and throws exception on calls for a files.
So we need another way to get cotent from ipfs hash.

We will use it to get image content. Then we will convert it to base64 and show via src="data:image/*,base64,..."